src/dispatching/repls.rs

```diff
@@ -1,4 +1,11 @@
-//! REPLs for dispatching updates.
+//! [REPL]s for dispatching updates.
+//!
+//! This module provides functions for easy update handling, that accept a
+//! single "handler" function that processes all updates of a certain kind. Note
+//! that REPLs are meant to be used as a prototyping tool and lack configuration
+//! and some advanced features.
```

Comment:
```suggestion
//! This module provides utilities for easy update handling. They accept a
//! single "handler" function that processes all updates of a certain kind. Note
//! that REPLs are meant to be used for simple scenarios, such as prototyping, inasmuch they lack configuration
//! and some advanced features.
```

1. We use REPLs for real-world bots and not just for prototyping.
2. I suggest linking "... and some [advanced features]" to that section comparing dispatching and REPLs, somewhere in `src/dispatching.rs` if my memory serves me correctly.

Also, please do **not** commit these suggestions right from GitHub because you'll break formatting.